packages/netsuite-adapter/src/client/sdf_client.ts

```diff
@@ -912,8 +913,9 @@ export default class SdfClient {
   private static customizeDeployError(error: Error): Error {
     const errorMessage = error.message
     if (settingsValidationErrorRegex.test(errorMessage)) {
-      if (manifestErrorRegex.test(errorMessage)) {
-        return new ManifestValidationError(errorMessage)
+      const manifestErrorScriptidArray = getGroupItemFromRegex(errorMessage, manifestErrorDetailsRegex, OBJECT_ID)
```

Comment:
nit (and maybe my personal preference):
```suggestion
      const manifestErrorScriptids = getGroupItemFromRegex(errorMessage, manifestErrorDetailsRegex, OBJECT_ID)
```